src/Containers/StoreHoursList/CreateHoursModal.tsx

```diff
@@ -13,223 +15,310 @@ import { Select } from '../../Inputs/Select/Select';
 import { MainInterface, ResponsiveInterface } from '../../Utils/BaseStyles';
 import { Mixins } from '../../Utils';
 
+const NO_MERGED_HOURS = 0;
+const NO_UNMERGED_HOURS = 0;
+
 interface CreateHoursProps
     extends MainInterface,
         ResponsiveInterface,
         React.HTMLAttributes<HTMLDivElement> {
     isVisible: [boolean, React.Dispatch<React.SetStateAction<boolean>>];
     MODAL_HEADER: string;
     SELECT_A_DAY_TITLE: string;
-    fromTimeTooBigError: string;
-    toTimeTooSmallError: string;
     SELECT_A_CATEGORY: string;
     ADD_HOURS_BUTTON: string;
-    errorMessage: string;
+    FROM_TIME_TO_BIG: string;
```

Comment:
Why not just have on error message?

Reply:
There is a seperate error message for each I assume to be more customizable, though this was prior to my working on it.

Combining these to an object might make sense but it would be different than the rest of the textHeaders errors so I think it would be okay to leave it this way.

Reply:
That doesn't make sense becuase if you have different errors you won't have 1 prop per Type of Error.